packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -0,0 +1,298 @@
+import {
+  IComboBoxStyles,
+  IComboBoxOptionStyles,
+  IComboBoxCaretDownButtonStyles
+} from './ComboBox.Props';
+import {
+  ITheme,
+  IStyle,
+  mergeStyleSets,
+  FontSizes,
+  FontWeights,
+  getFocusStyle
+} from '../../Styling';
+import { memoizeFunction } from '../../Utilities';
+
+const MS_HIGHCONTRAST_ACTIVE = '@media screen and (-ms-high-contrast: active)';
+const MS_HIGHCONTRAST_BLACK_ON_WHITE = '@media screen and (-ms-high-contrast: black-on-white)';
+
+const ComboBoxHeight = '32px';
+const ComboBoxLineHeight = '30px';
+const ComboxBoxCaretDownWidth = '32px';
+const ComboBoxOptionHeight = '36px';
+
+const getDisabledStyles = memoizeFunction((theme: ITheme): IStyle => {
+  const { semanticColors, palette } = theme;
+  return {
+    backgroundColor: semanticColors.disabledBackground,
+    borderColor: semanticColors.disabledBackground,
+    color: semanticColors.disabledText,
+    cursor: 'default',
+    [MS_HIGHCONTRAST_ACTIVE]: {
+      borderColor: 'GrayText',
+      color: 'GrayText'
+    },
+
+    [MS_HIGHCONTRAST_BLACK_ON_WHITE]: {

```

Comment:
can remove the HC styles specific to black-on-white since the hc-active blocks will cover it